From 9f6a877067ccec3e0b329828bf0a211ae797a1e1 Mon Sep 17 00:00:00 2001
From: Cesare Mercurio <cesare.mercurio@gmail.com>
Date: Sat, 16 Dec 2023 22:50:01 -0800
Subject: [PATCH 1/2] VVI-OpenCL-CLHPP : Patch to add MODULE.bazel

Summary:
Make it bzlmod compatible

Test Plan:
integration with vvplatform
---
 MODULE.bazel | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 MODULE.bazel

diff --git a/MODULE.bazel b/MODULE.bazel
new file mode 100644
index 0000000..863db91
--- /dev/null
+++ b/MODULE.bazel
@@ -0,0 +1,6 @@
+module(
+    name = "opencl-clhpp",
+    version = "2023.12.14",
+    compatibility_level = 1,
+)
+bazel_dep(name = "rules_cc", version = "0.0.5")
\ No newline at end of file
-- 
2.34.1

